Assets/Scripts/Player/PlayerController.cs

```diff
@@ -16,37 +12,37 @@ public class PlayerController : MonoBehaviour {
         this.floorMask = LayerMask.GetMask ("Floor");
     }
 
-    void Start () {
-        this.playerRigidbody = GetComponent<Rigidbody> ();
+    protected override void Start () {
+        base.Start();
         this.inventory = GetComponent<PlayerInventory> ();
+        baseStats [Stat.Speed] = 5.0f;
```

Comment:
Oh and I did notice StatValues and the similarity to Modifiers. I wasn't trying to clobber your naming scheme or architecture; it's just that in my head there's a difference between a base stat and a modifier? If that makes sense? Even though they are both a Stat & Value tuple.